dvc/data/raw/ner.dvc

```diff
@@ -0,0 +1,6 @@
+wdir: ../../../data/raw
+outs:
+- md5: 43d5c9ca303e4eccba73044b10eef279.dir
+  size: 262617
+  nfiles: 1
+  path: ner
```

Comment:
Doesn't look like any script is reading files from `raw/ner`. What is the purpose of this file?

Reply:
Old files from when I was experimenting prior to you setting up the work flow. Deleted.